src/Support/GeometryProxy.php

```diff
@@ -163,11 +163,7 @@ private function determineCoordinateIndices($index, $length)
      */
     protected function exposeRawIfAvailable($argument)
     {
-        if (method_exists($argument, 'getRawGeometry')) {
-            return $argument->getRawGeometry();
-        }
-
-        return $argument;
+        return (method_exists($argument, 'getRawGeometry')) ? $argument->getRawGeometry() : $argument;
```

Comment:
I would rather have cleaner code than an extra 1% coverage, so keep the old.

I understand why you would push for 100 though, it's addictive!